.ci/linux-steps.yaml

```diff
@@ -44,7 +44,12 @@ steps:
 
     # Install cereal.
     wget https://github.com/USCiLab/cereal/archive/v1.3.0.tar.gz
-    tar -xvzpf v1.3.0.tar.gz # Unpack into cereal-1.3.0/.
+        tar -xvzpf v1.3.0.tar.gz # Unpack into cereal-1.3.0/.
+        cd cereal-1.3.0 && \
+        cmake . && \
+        make && \
```

Comment:
You can probably get away without the `make` here---since cereal is header-only, I guess we don't need to build anything.  In fact, you could even do something just like `cp -vr include/cereal /usr/include/` or something like that.  I think that will help with the Linux build errors, at least. :)